lib/lambda_ethereum_consensus/p2p/incoming_requests/handler.ex

```diff
@@ -28,18 +28,9 @@ defmodule LambdaEthereumConsensus.P2P.IncomingRequests.Handler do
   @spec handle_req(String.t(), String.t(), binary()) ::
           :ok | :not_implemented | {:error, binary()}
   defp handle_req("status/1/ssz_snappy", message_id, message) do
-    # hardcoded response from random peer
-    current_status = %SszTypes.StatusMessage{
-      fork_digest: Base.decode16!("BBA4DA96"),
-      finalized_root:
-        Base.decode16!("7715794499C07D9954DD223EC2C6B846D3BAB27956D093000FADC1B8219F74D4"),
-      finalized_epoch: 228_168,
-      head_root:
-        Base.decode16!("D62A74AE0F933224133C5E6E1827A2835A1E705F0CDFEE3AD25808DDEA5572DB"),
-      head_slot: 7_301_450
-    }
-
-    with <<84, snappy_status::binary>> <- message,
+    with {:ok, current_status} <-
+           LambdaEthereumConsensus.ForkChoice.Helpers.current_status_message(),
```

Comment:
should be more like `LambdaEthereumConsensus.ForkChoice.Store.current_status_message()`

Reply:
It is not related about the incoming message in the first glance we fetch the node's current status so I believe this change is not related